doc/interactors.md

```diff
@@ -286,6 +287,35 @@ RadioButton("Label").exists();
 - `hasWarning`: _boolean_ = `true` if the radio button has a warning [class]
 - `hasError`: _boolean_ = `true` if the radio button has an error [class]
 
+#### SearchField
+
+Stripes SearchField component composes basic `<select>` and `<input>` elements
+
+##### Synopsis
+
+``` javascript
+import { SearchField } from '@folio/stripes-testing';
+
+SearchField().has({ id: 'searchFieldTest'});
+```
+
+##### Actions
+
+- `selectIndex(string)`: selects the option matching option name
+- `fillIn(value: string)`: fills in the text field with a given value
+
+##### Filters
+
+- `id`: _string_ = the DOM element id of the contained `select`
+  element.  Note that this _not the id of the React component_. The `id`
+  filter is provided for debugging, but should not generally be used
+  in tests since it is not a user-facing value.
+- `placeholder`: _string_ = The placeholder text of the contained `input` element.
+- `value`: _string_ = the `value` property of the contained `input` element.
+- `readOnly`: _boolean_ = `true` if the contained `input` is read-only.
+- `isDisabledIndex`: _boolean_ = `true` if the contained `select` is disabled.
+- `isDisabled`: _boolean_ = `true` if the contained `input` is disabled.
```

Comment:
let's just have a single filter `disabled: boolean` since that is really the only user facing property there is on the component. When testing that the `select` is disabled, that only happens when the search field is loading, and so needing to make assertions on this only needs to happen in the context of the stripes-components test suite and we can use `interactor.assert()` to handle this one-off.